pkg/flowkit/config/json/emulator.go

```diff
@@ -46,12 +47,18 @@ func (j jsonEmulators) transformToConfig() (config.Emulators, error) {
 
 	return emulators, nil
 }
+func emulatorIsDefault(emulator config.Emulator) bool {
+	return reflect.DeepEqual(emulator, config.DefaultEmulator())
+}
 
 // transformToJSON transforms config structure to json structures for saving.
 func transformEmulatorsToJSON(emulators config.Emulators) jsonEmulators {
 	jsonEmulators := jsonEmulators{}
 
 	for _, e := range emulators {
+		if emulatorIsDefault(e) {
```

Comment:
```suggestion
		if e == config.DefaultEmulator() {
```

Reply:
Wouldn't this be ok? Reflection should be avoided if not really necessary.

Reply:
addressed